pages/common/kosmorro.md

```diff
@@ -0,0 +1,20 @@
+# kosmorro
+
+> Compute the ephemerides and the events for a given date, at a given position on Earth.
+> More information: http://kosmorro.space.
```

Comment:
We forgot to add the `<>` to the URL. Also, the URL redirects to the official repo. Maybe we can use the github URL directly.
```suggestion
> More information: <https://github.com/Deuchnord/kosmorro>.
```

Reply:
> Maybe we can use the github URL directly.

I'm not quite sure about that, this redirection is temporary, I'm making a website to replace it in the future. Or maybe I should make a PR when it'll be done?